service/templates/base_angular.html

```diff
@@ -20,6 +20,13 @@
         <!-- our css -->
         <link href="/static/dist/css/styles.min.css?hash={{ BuildRevision }}" rel="stylesheet" />
 
+        {{if .Bugsnag}}
+        {{if ne .Bugsnag ""}}
+        <script src="//d2wy8f7a9ursnm.cloudfront.net/v6/bugsnag.min.js"></script>
+        <script>window.bugsnagClient = bugsnag('.Bugsnag')</script>
```

Comment:
Isn't this supposed to be the API token? It's not currently injected into the template

Reply:
sorry, fixed